dask/base.py

```diff
@@ -23,6 +23,41 @@ class Base(object):
 
     def visualize(self, filename='mydask', format=None, optimize_graph=False,
                   **kwargs):
+        """
+        Render the computation of this object's task graph using graphviz.
+
+        Requires `graphviz` to be installed.
```

Comment:
On Mon, Jun 6, 2016 at 4:49 PM Jim Crist <notifications@github.com> wrote:

> In dask/base.py
> <https://github.com/dask/dask/pull/1234#discussion_r65975988>:
>
> > @@ -23,6 +23,41 @@ class Base(object):
> >
> >      def visualize(self, filename='mydask', format=None, optimize_graph=False,
> >                    **kwargs):
> > +        """
> > +        Render the computation of this object's task graph using graphviz.
> > +
> > +        Requires `graphviz` to be installed.
>
> The way I understood it was single backticks are only good when
> referencing variables to the function by their name, everything else should
> be double backticks. Either way, sphinx often doesn't do code-highlighting
> for things in single backticks, and we end up with odd italics everywhere
> instead. This may be an issue with our sphinx config though.
>

Ok I'll fix them, but this is what I was thinking of

https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#common-rest-concepts


> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dask/dask/pull/1234/files/140e7d97eaca069dda90e0248a01c973457b4285#r65975988>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AASE5Ji93N93FDgtELgL-FzYe4dW9fBeks5qJJYFgaJpZM4IvS13>
> .
>